src/helpers/attrValidation.js

```diff
@@ -148,3 +148,12 @@ export const getAttrValidators = (validationRules) => {
 
   return validators;
 };
+
+export const linkValidate = memoize(langCode => input => (
+  !input.value.symbolicDestination || !input.values[langCode]
```

Comment:
Hi @raxkaynan @gidesan , this code is `validating` that the link value and text both are not null. But Admin Console gives validation error when you hit `save` and BE is reporting an error. So, if we want parity with Admin Console then I am suggesting to remove checking of `!input.values[langCode]` (a.k.a text). 
Also, the jira related to this commit seems to be `done` so maybe we should re-consider merging this code at all? https://jira.entando.org/browse/ENG-384

Reply:
@ichalagashvili , I'm not sure why the Jira ticket has been moved to done as I tested it and the issue is still there. I'll check with QE guys and get back to you about this.

Re: validation, I think it's supposed to be validated on the FE when the attribute is required so that it prevents the form from being saved (i.e. disabled button) when values are empty.